ci-jobs/scripts/build-webdriveragent.js

```diff
@@ -29,12 +29,27 @@ async function buildWebDriverAgent (xcodeVersion) {
   await mkdirp(uncompressedDir);
   log.info('Creating tarball');
 
-  // Moved DerivedData/WebDriverAgent-* from Library to folder
+  // Move contents of this folder to uncompressed folder
+  await exec('rsync', [
+    '-av', rootDir, uncompressedDir,
+    '--exclude', path.resolve(rootDir, 'node_modules'),
+    '--exclude', path.resolve(rootDir, 'build'),
+    '--exclude', path.resolve(rootDir, 'ci-jobs'),
+    '--exclude', path.resolve(rootDir, 'lib'),
+  ]);
+
+  // Moved DerivedData/WebDriverAgent-* from Library to uncompressed folder
   const derivedDataPath = path.resolve(os.homedir(), 'Library', 'Developer', 'Xcode', 'DerivedData');
   const wdaPath = (await fs.glob(`${derivedDataPath}/WebDriverAgent-*`))[0];
   await mkdirp(path.resolve(uncompressedDir, 'DerivedData'));
   await fs.rename(wdaPath, path.resolve(uncompressedDir, 'DerivedData', 'WebDriverAgent'));
 
+  // Remove unneeded directories
+  const unneededDirectories = ['node_modules', 'ci-jobs', 'test', 'lib', 'build'];
```

Comment:
if we've already excluded these in the rsync, do we need to rm them here, or is this just a double check?

Reply:
Ahhh yes good catch. I forgot to get rid of that.